### PR TITLE
[cker] Introduce optimized kernels for quantization

### PR DESCRIPTION
**Pull request by ragmani**
This commit introduces optimized kernels for quantization.

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

---
For #10002
Draft #10329

### CODE REVIEW

File: compute/cker/include/cker/operation/Quantize.h [hunk 1]
```diff
@@ -45,6 +46,164 @@ inline void Quantize(const Shape &input_shape, const InputT *input_data, const S
...
+#endif // NEON
+
+  for (; i < flat_size; ++i)
+  {
+    const float val = input_data[i];
+    const int32_t unclamped = static_cast<int32_t>(round(val / scale)) + zero_point;
+    const int32_t clamped = std::min(std::max(unclamped, min_val), max_val);
+    output_data[i] = clamped;
+  }
+}
```
Comment (ragmani) [inline compute/cker/include/cker/operation/Quantize.h:205]: These functions are from tensorflow v2.10.

File: compute/cker/include/cker/operation/Round.h [hunk 2]
```diff
@@ -41,6 +42,26 @@ inline float RoundToNearest(float value)
...
+  static const float32x4_t point5_val_dup = vdupq_n_f32(0.5f);
+  static const float32x4_t minus_point5_val_dup = vdupq_n_f32(-0.5f);
+
+  const uint32x4_t mask = vcltq_f32(input, zero_val_dup);
+  const float32x4_t round = vbslq_f32(mask, minus_point5_val_dup, point5_val_dup);
+  return vcvtq_s32_f32(vaddq_f32(input, round));
+#endif // defined(__aarch64__) || defined(__SSSE3__)
+}
+
+#endif // NEON
```
Comment (ragmani) [inline compute/cker/include/cker/operation/Round.h:63]: These function is from tensorflow v2.10.

**Review by hseok-oh — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**